src/java/org/apache/cassandra/repair/RepairTask.java

```diff
@@ -26,6 +26,11 @@
 {
     String name();
 
+    default String successMessage()
```

Comment:
this is a pet-peeve I have had for a while... Preview repair does a Notification of the success message, and Notifications are lossy, so the user calling repair may not see it, by plugging it in as a part of the repair task, we can make sure preview repair can tack on the actual message which then gets saved and exposed to the table